GRAPH-1542 : TLS Examples for Aerospike Graph <-> Aerospike DB and Aerospike Graph <-> Gremlin (#34)
aerospike/aerospike-graph: tls_examples/AGS_AeroDB/README.md

Reviewer: Would suggest `AGS_AerospikeDB.md` for the file name.

> # Transport Layer Security Between AGS and Aerospike Server
> 
> This directory is an example on how to setup and use the Transport Layer Security feature
> with Aerospike Graph. Including creating new docker containers for Aerospike Graph Service (AGS) and Aerospike Server.

Reviewer: ```suggestion
# TLS Between AGS and Aerospike Server

This example will cover how to configure and use TLS between Aerospike Graph Service and Aerospike Server. This includes creating a new docker container for Aerospike Graph Service (AGS) and Aerospike Server.
```

> 
> In this example, we will setup TLS between AGS and Aerospike Server.

Reviewer: ```suggestion
```

## How it Works

### Cluster Name Match Setup
All of your Aerospike server nodes must use the same certificate.
The certificate must include the cluster name defined in CN field such as:
```
Subject: C=US, ST=California, L=San Jose, O=Acme, Inc., OU=Aerospike Cluster, CN=as-cluster-west
```

All Aerospike server nodes must use the same aerospike.conf file, with 
the same cluster name specified in the aerospike.conf file.

A tls sub-stanza for ```<cluster-name>``` has to be defined in the aerospike.conf 
file and the tls-name in the service sub-stanza must be specified and have 
the value ```<cluster-name>``` in order for the node’s configured cluster name to 
be used as the TLS name.
Using the cluster name (cluster-name) requires using Aerospike’s heartbeat-v3 protocol.

On client applications, for each seed host passed into the ```cluster_create()```

> call, pass in the same ```<cluster-name>``` for the tls-name as the TLS name.

Reviewer: what does this mean?
